editor/metaboxes/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import './metabox-iframe.scss';
+import MetaboxIframe from './iframe.js';
+import { handleMetaboxReload, metaboxStateChanged } from '../actions';
+import { getMetabox, isSavingPost } from '../selectors';
+
+function Metabox( {
+	isActive,
+	isDirty,
+	isUpdating,
+	isPostSaving,
+	location,
+	isSidebarOpened,
+	id = 'gutenberg-metabox-iframe',
+	metaboxReloaded,
+	changedMetaboxState,
+} ) {
+	if ( isActive === false ) {
+		return null;
+	}
+
+	const classes = classnames(
+		'gutenberg-metabox-iframe',
+		location,
+		{ 'sidebar-open': isSidebarOpened }
+	);
+
+	return (
+		<MetaboxIframe
+			isActive={ isActive }
+			isDirty={ isDirty }
+			isUpdating={ isUpdating }
+			isPostSaving={ isPostSaving }
+			title={ __( 'Extended Settings' ) }
+			key="metabox"
+			id={ id }
+			className={ classes }
+			location={ location }
+			metaboxReloaded={ metaboxReloaded }
+			changedMetaboxState={ changedMetaboxState }
+			src={ `${ window._wpMetaboxUrl }&metabox=${ location }` } />
```

Comment:
So do a double Connect? I was figuring we could just use the upper component to talk to Redux and leave this be, even if we are bubbling up. I see that as a good thing. We can't merge the two into one component as we won't be able to reliably grab a ref when the component mounts, as it might mount without the iframe existing. This could definitely be consolidated if we use `createEditorInstance` later. A lot of this extra layer of hacks could be removed in that case.